dashboard/app/views/pd/application/facilitator_application/new.html.haml

```diff
@@ -0,0 +1,20 @@
+= stylesheet_link_tag 'css/pd', media: 'all'
+%script{src: minifiable_asset_path('js/pd/application/facilitator_application/new.js'), data: @script_data}
+
+%h1
```

Comment:
Any reason this is not part of the react element? Is it just because it's completely static?

I feel like for some components we've put everything in the react element. If we have a standard / preferred way of doing this, lets follow it.

Reply:
Right, it's static and also rendered across all pages in the form. The React Form renders its own pages and doesn't have space for a header so I'd have to create a wrapper component with this header. It just seems like unnecessary overhead for some static content.

Reply:
I don't think we have a standard. Anything with client-side logic should be in React, but otherwise we tend to leverage rails & haml for surrounding static content so I think this is consistent.